content/07-native-tokens/03-getting-started.mdx

```diff
@@ -49,49 +49,50 @@ Once you have started the node, leave it running in its own terminal window (or
 Once a relay node has been set up, the `cardano-cli` command can be used to interact with the network as usual: 
 
 ```
-cd mary-ppe
+cd testnet
 cardano-cli ...
 ```
 
-Note that multi-asset support is provided as part of the Mary consensus era, so many commands will require the `--mary-era` flag:
+### Understanding values
+
+cardano-node version 1.35.0 and later of the cli no longer supports ASCII token names, only hex-encoded format is supported. The result of running  
```

Comment:
As per my previous comment. I suggest changing the version to 1.32.1 because this isn't a new change in 1.35.0